test/sandbox.html

```diff
@@ -19,6 +19,7 @@
       input: 'text',
       icon: 'success',
       showCloseButton: true,
+      showDenyButton: true,
```

Comment:
Is it intentional to set `showDenyButton` to `true` in the sandbox page? Is it to showcase the new deny button?

Reply:
Yes, for now let's have it in the sandbox so we can play with it, e.g. I'm planning to add some styles and scss vars so users will be able to rearrange the order in their styles and also put space between buttons (e.g. two buttons on the left side, one on the right side)